install/gcp/deployment_manager/README.md

```diff
@@ -0,0 +1,12 @@
+# Google Deployment Manager Template
+
+This directory contains a Google Cloud Deployment Manager template for getting
+up-and-running with a Google Cloud Kubernetes Engine cluster with Istio
+included.
+
+If you have the Google Cloud SDK installed (get it [here](https://cloud.google.com/sdk/)), you can create a new deployment via the command:
+```
+$ gcloud deployment-manager deployments create my-istio-deployment --config=istio-cluster.yaml
+```
+
```

Comment:
may want to add that you need to have the specific IAM role:

>> NOTE: You must set your default compute service account to include:

- ```roles/container.admin``` (Container Engine Admin)
- ```Editor``` (on by default)

To set this, navigate to the IAM section of the Cloud Console and find your default GCE/GKE service account in the following form to set that permission:

```projectNumber-compute@developer.gserviceaccount.com```



also, a user can invoke the template directly and pass parameters, (it just saves a small step):

gcloud deployment-manager deployments create istio2 --template https://raw.githubusercontent.com/...path/to/hosted/istio-cluster.jinja  --properties enableMutualTLS:false,gkeClusterName:istio-gke

Reply:
Good call! Added.